src/pve_exporter/cli.py

```diff
@@ -107,7 +109,14 @@ def main():
         with open(params.config) as handle:
             config = config_from_yaml(yaml.safe_load(handle))
 
+    gunicorn_options = {
+        'bind': f'{params.address}:{params.port}',
+        'threads': 2,
+        'keyfile': params.__dict__['server.keyfile'],
+        'certfile': params.__dict__['server.certfile'],
```

Comment:
There is a way to avoid using `params.__dict__` here: Add a `dest` argument in the `parser.add_argument` call above in order to specify the key name. I suggest the following key names:

```
parser.add_argument('--server.keyfile', dest='server_keyfile'
                    help='SSL key for server')
parser.add_argument('--server.certfile', dest='server_certfile'
                    help='SSL certificate for server')
```

This way the values can be accessed using the property syntax.

```
    'keyfile': params.server_keyfile,
    'certfile': params.server_certfile,
```